server/s3_object_data_copier.cc

```diff
@@ -362,6 +371,47 @@ void S3ObjectDataCopier::copy(
   s3_log(S3_LOG_DEBUG, "", "%s Exit", __func__);
 }
 
+void S3ObjectDataCopier::copy_part_fragment_in_single_source(
```

Comment:
We will have >1 parts from source multipart object which are getting copied in target bucket as single part. So, the function name should be something like `copy_part_fragments_in_single_target_part()` or `copy_multipart_object_in_single_target_part()`